src/backend/TrafficCourts/Staff.Service/Mappers/Mapper.cs

```diff
@@ -61,9 +61,9 @@ public static DisputeRejected ToDisputeRejected(Dispute dispute)
         return disputeRejected;
     }
 
-    public static EmailSendValidation ToEmailSendValidation(Guid uuid, string host)
+    public static EmailVerificationSend ToEmailSendValidation(Guid uuid, string host)
     {
-        EmailSendValidation emailSendValidation = new(uuid, host);
+        EmailVerificationSend emailSendValidation = new(uuid, host);
         return emailSendValidation;

```

Comment:
I think this should be "emailVerificationSend" instead of "emailSendValidation" as well.

Reply:
Good find. I'll rename this in a subsequent PR I have with other renaming.